src/main/java/ua/com/javarush/oleksandr/reddit/redditcloneabstract/dto/PostRequest.java

```diff
@@ -15,9 +11,7 @@
 @NoArgsConstructor
 @EqualsAndHashCode(of = "url")
 @Builder(builderMethodName = "with")
-public class PostRequestDTO {
-
-    private Long id;
+public class PostRequest {
```

Comment:
I think keeping it as PostResponseDto is clearer. The 'DTO' suffix lets me know right away that it's a Data Transfer Object. It's particularly useful when I'm working with code I'm not familiar with. But I'm okay with changing the name, just remark to think about.

Reply:
Maybe it's just that to me the dto suffix at the end doesn't look very pretty for a class name